### PR TITLE
updated:Readme.md example table updated

### PR DESCRIPTION
**Pull request by anupshrestha11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,19 @@ Example:
             <td>1978</td>
             <td>100</td>
           </tr>
+          <tr>
+            <td>Micheal</td>
```
Comment (LeeWannacott) [inline README.md:45]: Minor typo: Micheal is spelled Michael in Michael Jackson's name.

File: README.md [hunk 2]
```diff
@@ -41,6 +41,19 @@ Example:
...
+            <td>Jackson</td>
+            <td>1964</td>
```
Comment (LeeWannacott) [inline README.md:47]: MJ was born 1958. For dates to showcase what table sort can do, I think we can change Date to the format: 1958/8/29 and use different delimiters for the different people e.g 1958-8-29 1958.8.29 etc. . So if we change all the dates in the table to this format and perhaps to real people: like for example changing Bach Frank to Benjamin Franklin or Sebastian Bach then you can put a real date.

Ben 10 is fine though can just make up a date for him; maybe when the show started?, or something.

**Review by LeeWannacott — CHANGES_REQUESTED** [review]
If you could change date format and add some real people; so then you can add some real dates that would be great :+1: 

Like for example:
table-screen

**Comment by anupshrestha11** [comment]
I will make changes to the table.

**Comment by LeeWannacott** [comment]
Thanks. Commit landed in 387ef05

**Pull request merged**